internal/api/web/views/pages/dashboard_pages/dashboard_home_page.templ

```diff
@@ -0,0 +1,36 @@
+package dashboard_pages
+
+import (
+	"github.com/Molnes/Nyhetsjeger/internal/api/web/views/components/layout_components"
+	"github.com/Molnes/Nyhetsjeger/internal/data/quizzes"
+	"github.com/Molnes/Nyhetsjeger/internal/api/web/views/components/dashboard_components/dashboard_home_page"
+)
+
+// Dashboard home page
+templ DashboardHomePage(unpublishedQuizzes []quizzes.Quiz, publishedQuizzes []quizzes.Quiz) {
+	@layout_components.DashBoardLayout() {
+		<div class="flex flex-col pl-8 pt-8 pr-48">
```

Comment:
Set even horizontal padding

Reply:
7609db67f80a09a15a1c1546103417c518501aa8
not sure if this looks good